hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/CallerContext.java

```diff
@@ -47,6 +47,7 @@ public final class CallerContext {
   // field names
   public static final String CLIENT_IP_STR = "clientIp";
   public static final String CLIENT_PORT_STR = "clientPort";
+  public static final String DFSROUTER_PORT_STR = "dfsrouterPort";
```

Comment:
@LiuGuH if this comment is addressed, then also things look good IMO. 
The point is both `dfsrouterPort` and using `isClientPortInfoAbsent()` to set it in the audit logs, together seems fragile because anyone in future passes clientPort in the context and will get dfsrouterPort printed, which is not correct for non-router usecases.